<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppSettingsDialog</class>
 <widget class="QDialog" name="AppSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Outputs</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt;DViz supports multiple outputs for various uses.&lt;br /&gt;Use this button to setup options for outputs.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnConfigOutputs">
        <property name="text">
         <string>Configure Live Outputs</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Graphics Acceleration</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cbUseOpenGL">
        <property name="text">
         <string>Use OpenGL Graphics Acceleration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;img src=&quot;:/data/opengl-logo.png&quot; align=&quot;right&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt;The only reason to turn off OpenGL acceleration is if &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt;the program doesn't run smoothly or freezes with &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt;OpenGL turned on.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Cache Size</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>DViz stores the results of certain graphical operations 
in memory inorder to speed up the display of slides. 
Less memory allocated to that cache means that DViz
potentially has to redraw items more frequently, 
resulting in slower performance.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Cache Size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="cacheSlider">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>1024</number>
          </property>
          <property name="singleStep">
           <number>16</number>
          </property>
          <property name="pageStep">
           <number>64</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>128</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="cacheBox">
          <property name="suffix">
           <string> MB</string>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>1024</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Cross Fade Speed &amp;&amp; Quality</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Speed is how long the total fade should take, expressed
in milliseconds. There are 1000ms in 1 second.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Speed</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Quality</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="speedSlider">
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>5000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>500</number>
          </property>
          <property name="value">
           <number>250</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>500</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="qualitySlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>60</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>15</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="speedBox">
          <property name="suffix">
           <string> ms</string>
          </property>
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>5000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="qualityBox">
          <property name="suffix">
           <string> frames</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>60</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Quality is the number of intermediate (partially faded)
frames to show from start to end of the cross fade. 
The more frames, the slower the fade may run and the
slower videos may display.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AppSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>668</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AppSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>668</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speedSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>speedBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>506</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speedBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>speedSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>517</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qualitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>qualityBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qualityBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>qualitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>549</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>533</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cacheBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cacheSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cacheSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cacheBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
